dataservice/api/common/enumerations.yaml

```diff
@@ -0,0 +1,165 @@
+# How should values be cased - lower case, proper/title case?
+# What should default value be if Not Reported in source data?
+
+study:
+    release_status:
+        choices:
+            - Pending # Not sure about the choices here
+        default: Pending
+
+study_file:
+    availability:
+        choices:
+            - Immediate Download
+            - Cold Storage
+        default: Immediate Download
+
+participant:
+    gender:
+        choices:
+            - Male
+            - Female
+            - Unknown
+            - Not Reported
+        default: Not Reported
+    race:
+        choices:
+          - White
+          - American Indian or Alaska Native
+          - Black or African American
+          - Asian
+          - Native Hawaiian or Other Pacific Islander
+          - Other
+          - Unknown
+          - Not Reported
+        default: Not Reported
+    ethnicity:
+        choices:
+            - Hispanic or Latino
+            - Not Hispanic or Latino
+            - Unknown
+            - Not Reported
+        default: Not Reported
+
+diagnosis:
+    diagnosis_category:
+        choices:
+            - Structural Birth Defect
+            - Cancer
+            - Other
+            - Unknown
+        default: Other
+
+phenotype:
+    observed:
+        choices:
+            - Positive
+            - Negative
+            - Unknown
+        default: Unknown
+
+outcome:
+    vital_status:
+        choices:
+            - Alive
+            - Dead
+            - Unknown
+            - Not Reported
+        default: Not Reported
+    disease_related:
+        choices:
+            - Yes
+            - No
+            - Unknown
+            - Not Reported
+        default: Not Reported
+
+biospecimen:
+    analyte_type:
+        choices:
+            - DNA
+            - RNA
+            - Unknown
+            - Not Reported
+        default: Not Reported
+
+genomic_file:
+    data_type:
+        choices:
+            - Submitted Aligned Reads
+            - Submitted Aligned Reads Index
+            - Simple Nucleotide Variation
+            - Other
+        default: Other
+    file_format:
+        choices:
+            - ???
+        default: ???
+    availability:
+        choices:
+            - Immediate Download
+            - Cold Storage
+        default: Immediate Download
+
+sequencing_experiment:
+    experiment_strategy:
+        choices:
+            - WGS
+            - WXS
+            - RNA-Seq
+            - ChIP-Seq
+            - miRNA-Seq
+            - Bisulfite-Seq
+            - Validation
+            - Amplicon
+            - Targeted Sequencing
+            - Other
+            - Not Reported
+        default: Not Reported
+    platform:
+        choices:
+            - Illumina
+            - SOLiD
+            - LS454
+            - Ion Torrent
+            - Complete Genomics
+            - PacBio
+            - Other
+            - Not Reported
+        default: Not Reported
+    instrument_model:
```

Comment:
Do we need an exhaustive list here? There's quite a few models out there

Reply:
Agreed, can we just start with the ones have been / will be used for the next year? We can gather those and then expand this list as needed in the future?